java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/jdbc/JdbcDialect.java

```diff
@@ -61,7 +61,7 @@ public interface JdbcDialect {
     Optional<String> getUpsertStatement(
             SchemaTableName schemaTableName, List<String> fieldNames, List<String> uniqueKeyFields);
 
-    void bindUpsertStatement(
+    void bindInsertOrUpsertStatement(
```

Comment:
`upsert` means `insert or update`. I think we don't need to change the name.

To handle insert for append only sink, we can simply call the `bindInsertStatement`.